CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 3.7.0
+
+-> SDK 2.3.0
+
+-   fix: POST /api/v2/Attributes will no longer accept RelationshipAttributes, as this was unintentional and led to wrong behavior
+-   add usecases for attribute succession
+-   refactor existing attribute use cases
```

Comment:
the user will never see a refactoring - this is no information for the user and should be removed or better specified.

Reply:
(and we have no "use cases" in the connector)